.github/workflows/javascript.yml

```diff
@@ -17,8 +17,12 @@ jobs:
         module:
           - name: 'mysql2'
             with_oceanbase_container: true
+          - name: 'drizzle'
+            with_oceanbase_container: true
+            oceanbase_image_tag: '4.2.3_BETA'
```

Comment:
only `oceanbase-ce >= 4.2.3` supports `serial` data type and that is required for Drizzle.